scripts/generateConstants.js

```diff
@@ -26,7 +26,7 @@ const renderFileContent = iconNames => `/**
 exports.VALID_SHAPES = [${iconNames.map(name => `'${name}'`)}];
 `;
 
-fs.writeFileSync(
+fs.appendFileSync(
```

Comment:
`write` will open a file and write to it; `append` will write the file if it doesn't exist. This is necessary because we actually clean `dist/` dirs now.